### ISSUE TITLE
Turn Wiki for Repo into User Guide

### ISSUE THREAD

**Issue by egrace479** [Imageomics/Andromeda#73]
Andromeda needs a User Guide!

We should describe the project:
- [ ] What Andromeda is compared to the data fetching. 
- [ ] Data limitations (starts getting crowded as data point numbers increase, especially with images).
- [ ] What is QUEST?
- [x] Different versions/releases of Andromeda (eg., QUEST 2023, latlonCover, Quest 2024) with descriptions and links to that particular version.
- [x] Why some columns might be dropped and what to do about it (eg., #68)
- [ ] Fetch data information:
    - [ ]  `latlonCover` info (including eg., #71)
    - [ ] SatRGB and general extra info (upcoming).

**Comment by egrace479** [comment]
Once this is properly set up, we'll want to add links to the documentation from the README in the repo and in the HF repo:
- [ ] Link to Docs in GH README
- [ ] Link to Docs in HF README

**Comment by johnbradley** [comment]
Let's add a link from the App to the documentation too.

**Comment by egrace479** [comment]
> Let's add a link from the App to the documentation too.

As in this note on the GH README?
> For instructions on building and running the Docker container see [Developing with Docker](https://github.com/Imageomics/Andromeda/wiki/Developing-with-Docker).

**Comment by johnbradley** [comment]
No. Right now this is the main Andromeda screen running on huggingface:
<img width="1066" alt="Screenshot 2024-02-22 at 10 17 47 AM" src="https://github.com/Imageomics/Andromeda/assets/1024463/90e157c0-913d-45dc-9c54-1317c84ba318">
My thought is users might visit the page and not realize we have a user guide. So my suggestion is to add a link to this page that opens the User Guide.

**Comment by egrace479** [comment]
Ah, like with the data dashboard link to documentation. That seems like a good idea, though we still need to write most of the user guide.
![Screenshot 2024-02-22 at 10 38 27 AM](https://github.com/Imageomics/Andromeda/assets/38985481/ea362a8a-b4db-4021-b10b-ed5f6e4e4837)

**Comment by egrace479** [comment]
Thoughts on organization of our user/admin guide:
- Home page can give overview of Andromeda
- QUEST page to discuss what QUEST is (sections for 2023, 2024, and on to the future). This could focus on descriptions, then point to appropriate [Prior Configurations](https://github.com/Imageomics/Andromeda/wiki/Prior-Configurations) for more information on the technical details.
- GUI-type page? Something to cover the limitations of space and similar.
- Page with more details on Andromeda and the implementation:
    - Cover things like why some columns might be dropped, general column requirements.
- Fetch data page:
    - Options for getting data
    - Tools that can be/are applied (`latlonCover` & SatRGB)

**Comment by egrace479** [comment]
@johnbradley, think we should establish MkDocs documentation instead of keeping it in the wiki?

**Comment by johnbradley** [comment]
Since Andromeda is pretty mature I would just keep it in the wiki.
